src/test/java/org/akhq/controllers/TopicControllerTest.java

```diff
@@ -9,7 +9,7 @@
 import org.akhq.utils.ResultPagedList;
 import org.apache.kafka.common.config.TopicConfig;
 import org.junit.jupiter.api.*;
-
+import org.akhq.models.Record;
```

Comment:
I have compilation problems here, with ambiguous usage of Record (java.lang.Record and Record of package models). I import it explicitly.